rules/windows/process_creation/win_susp_tracker_execution.yml

```diff
@@ -0,0 +1,31 @@
+title: DLL Injection with Tracker.exe
+id: 148431ce-4b70-403d-8525-fcc2993f29ea
+description: This rule detects DLL injection and execution via LOLBAS - Tracker.exe
+author: Avneet Singh '@v3t0_'
```

Comment:
Hello @V3T0! Would you mind adding "oscd.community" to the author field?
It will help with the promotion of the initiative and engage more people in the next sprints.

Reply:
sure, I'll add it @yugoslavskiy :)